src/main/java/org/mule/tools/apikit/output/GenerationModel.java

```diff
@@ -37,15 +38,22 @@
   private final String verb;
   private final Action action;
   private final Resource resource;
-  private final String mimeType;
+  private String mimeType;
   private final String version;
   private final List<String> splitPath;
   private final ApikitMainFlowContainer api;
+  private boolean shouldIncludeMimeTypeInName = true;
```

Comment:
final

Reply:
If I set it as final with a default value, I won't be able to change it in the constructor (https://github.com/mulesoft/apikit-rest-scaffolder/blob/6babd8e84659c8dad93347d1701e2d8ac241dcaa/src/main/java/org/mule/tools/apikit/output/GenerationModel.java#L54)

An alternative would be set it final and set the default value in the other constuctors. What do you think?

Reply:
best to keep it final, and set default values.